tensor/stats/cluster/cluster.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cluster
+
+//go:generate core generate
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+
+	"cogentcore.org/core/base/indent"
+	"cogentcore.org/core/tensor"
+	"cogentcore.org/core/tensor/stats/stats"
+)
+
+// todo: all of this data goes into the datafs
+// Cluster makes a new dir, stuffs results in there!
+// need a global "cwd" that it uses, so basically you cd
+// to a dir, then cal it.
+
+// Node is one node in the cluster
+type Node struct {
```

Comment:
Why isn't this a `tree.Node`?

Reply:
again, will be datafs as that comment said.